crates/bevy_math/src/direction.rs

```diff
@@ -41,10 +41,10 @@ impl std::fmt::Display for InvalidDirectionError {
 /// A normalized vector pointing in a direction in 2D space
 #[derive(Clone, Copy, Debug, PartialEq)]
 #[cfg_attr(feature = "serialize", derive(serde::Serialize, serde::Deserialize))]
-pub struct Direction2d(Vec2);
-impl Primitive2d for Direction2d {}
+pub struct Dir2(Vec2);
```

Comment:
Should we add something like `#[doc(alias = "Direction2d")]` to make searchability in docs.rs a bit easier?